src/style.css

```diff
@@ -1 +1,75 @@
 /* Styles go here */
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  color: var(--primary-color);
+  --primary-color: #000;
+  --secondary-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+*,
+*:before,
+*:after {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover {
+  color: var(--secondary-color);
+}
+
+.nav__text {
+  padding: 22.5px 0 18.5px;
+}
+
+.is-active {
+  display: flex;
+  flex-direction: column;
+  color: var(--secondary-color);
+}
+
+.is-active::after {
```

Comment:
pseudo-element `.is-active::after` should be positioned absolute and .is-active should have position:relative